sdk/src/main/java/com/akkaserverless/javasdk/lowlevel/EventSourcedEntityHandler.java

```diff
@@ -49,20 +45,15 @@
    * @param context The command context.
    * @return The reply to the command, if the command isn't being forwarded elsewhere.
    */
-  Reply<Any> handleCommand(Any command, CommandContext context);
+  EventSourcedEntityBase.Effect<Any> handleCommand(Any command, CommandContext context);
```

Comment:
Note that this effect is only half done as we haven't applied the events yet, and therefore the final reply message has not been created. Actual interpretation of this effect is in the `EventSourcedEntitiesImpl` stream (for now)